tensorflow/go/op/scope_test.go

```diff
@@ -95,6 +95,40 @@ func TestMultipleGeneratedOps(t *testing.T) {
 	}
 }
 
+func TestScopeWithGraph(t *testing.T) {
+	s1 := NewScope()
+
+	var hello interface{} = "hello"
+	tensor1, _ := tf.NewTensor(hello)
+	s1.AddOperation(tf.OpSpec{
```

Comment:
using the `Const` method leads to:
 scope_test.go:109: failed to add operation "Const": Duplicate node name in graph: 'Const'